src/Command/Docker/ConfigConvert.php

```diff
@@ -1,108 +0,0 @@
-<?php
-/**
- * Copyright © Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
- */
-namespace Magento\MagentoCloud\Command\Docker;
-
-use Magento\MagentoCloud\Docker\Config\Converter;
-use Magento\MagentoCloud\Filesystem\DirectoryList;
-use Magento\MagentoCloud\Filesystem\Driver\File;
-use Magento\MagentoCloud\Filesystem\FileSystemException;
-use Magento\MagentoCloud\Filesystem\SystemList;
-use Symfony\Component\Console\Command\Command;
-use Symfony\Component\Console\Input\InputInterface;
-use Symfony\Component\Console\Output\OutputInterface;
-
-/**
- * Converts raw array configuration to .env files.
- */
-class ConfigConvert extends Command
-{
-    const NAME = 'docker:config:convert';
```

Comment:
you remove this config at all?
if so, it is BIC. You need to leave this command and result of it should be output with explanation what to use instead

Reply:
It is a manual run command, which can be excluded from BC list

Reply:
hm. Manual command may be used in scripts by others to automate deployment on cloud docker, for example. And even if use it only manually, it is not good strategy to surprise people with absence of some commands after new release.
I agree with removing Extension in current moment as we do not provide BC for code itself, but removing/renaming commands or change their behavior is BIC IMHO.

Reply:
Added name alias